src/Files.App/Data/Parameters/GitConfirmLoginParams.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System.Text.Json.Serialization;
+
+namespace Files.App.Data.Parameters
+{
+	internal class GitConfirmLoginParams
+	{
+		[JsonPropertyName("client_id")]
+		public string ClientId { get; init; }
+
+		[JsonPropertyName("device_code")]
+		public string DeviceCode { get; init; }
+
+		[JsonPropertyName("grant_type")]
+		public string GrantType { get; init; }
+
+		public GitConfirmLoginParams(string clientId = "", string deviceCode = "") 
+		{
+			ClientId = clientId;
```

Comment:
Do I need to set the client id?

Reply:
In release no, we pass it as a parameter from GitHelpers, and there we retrieve it from the secret. Instead, if you want to test on your machine, you should set it